stellargraph/connector/neo4j/mapper.py

```diff
@@ -289,3 +293,82 @@ def sample_features(self, head_nodes, batch_num):
             )
 
         return features
+
+
+@experimental(reason="the class is not fully tested")
+class Neo4jClusterNodeGenerator(ClusterNodeGenerator):
```

Comment:
It's a bit weird to me that this is a subclass but doesn't call `super().__init__` and so has to duplicate everything here.